optics-core/src/Optics/Core/Extra.hs

```diff
@@ -0,0 +1,17 @@
+module Optics.Core.Extra
+  (
+    is
+  )
+  where
+
+import Optics.Internal.Optic
```

Comment:
Only `Optics.Optic` is needed, no Internal stuff is used here.

Reply:
Done